userspace/libscap/scap_procs.c

```diff
@@ -557,6 +557,11 @@ static int32_t scap_proc_add_from_proc(scap_t* handle, uint32_t tid, int parentt
 	}
 
 	//
+	// Gathers the exepath
+	//
+	snprintf(tinfo->exepath, target_res, "%s", filename);
```

Comment:
I got confused and used `filename` instead of `target_name` which holds the real exepath (after calling `readlink`).
I will change it and replace `target_res` with `sizeof(tinfo->exepath)` to be on the safe side.